geode-core/src/main/java/org/apache/geode/internal/cache/wan/parallel/ConcurrentParallelGatewaySenderEventProcessor.java

```diff
@@ -137,6 +140,35 @@ public void enqueueEvent(EnumListenerEvent operation, EntryEvent event, Object s
     this.processors[pId].enqueueEvent(operation, event, substituteValue);
   }
 
+  @Override
+  protected void registerEventDroppedInPrimaryQueue(EntryEventImpl droppedEvent) {
```

Comment:
I guess we don't need to call setModifiedEventId in the parallel case sine that isn't being used in the remove case. Right?